cmake/modules/SIMDExt.cmake

```diff
@@ -1,66 +1,60 @@
 # detect SIMD extentions
 #
-# ARM_NEON_FLAGS
-#
 # HAVE_ARMV8_CRC
-# HAVE_NEON
-# HAVE_SSE
-# HAVE_SSE2
-#
-# INTEL_SSE4_1
-# INTEL_SSE4_2
+# HAVE_ARMV8_SIMD
+# HAVE_ARM_NEON
+# HAVE_INTEL_SSE
+# HAVE_INTEL_SSE2
+# HAVE_INTEL_SSE3
+# HAVE_INTEL_SSSE3
+# HAVE_INTEL_PCLMUL
+# HAVE_INTEL_SSE4_1
+# HAVE_INTEL_SSE4_2
 #
-# SSE3_FLAGS
-# SSE4_FLAGS
+# SIMD_COMPILE_FLAGS
 #
 
 if(CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64|AARCH64")
   CHECK_C_COMPILER_FLAG(-march=armv8-a+crc HAVE_ARMV8_CRC)
   if(HAVE_ARMV8_CRC)
-    set(ARM_CRC_FLAGS "-march=armv8-a+crc -DARCH_AARCH64")
+    set(ARM_CRC_FLAGS "-march=armv8-a+crc")
   endif()
-  CHECK_C_COMPILER_FLAG(-march=armv8-a+simd HAVE_NEON)
-  if(HAVE_NEON)
-    set(ARM_NEON_FLAGS "-march=armv8-a+simd -DARCH_AARCH64 -DARM_NEON")
```

Comment:
ARM_NEON_FLAGS is only set for jerasure therefore modifying it has no impact elsewhere :+1: